flake.nix

```diff
@@ -19,10 +19,25 @@
         let
           pkgs = nixpkgsFor.${system};
         in {
+          # build bosh-bootloader CLI (bbl) locally because it is not available in the Nix package registry.
+          # this custom build can be removed once https://github.com/cloudfoundry/bosh-bootloader/issues/596 is implemented.
+          bosh-bootloader = pkgs.buildGoModule rec {
+            name = "bosh-bootloader";
+            src = pkgs.fetchgit {
+              url = "https://github.com/cloudfoundry/bosh-bootloader";
+              rev = "139141d8addd95ef381a29ef98b6ce2e9a78c437"; # commit hash of v9.0.17, see also https://github.com/cloudfoundry/bosh-bootloader/commit/139141d8addd95ef381a29ef98b6ce2e9a78c437
```

Comment:
Would be great to configure `renovate` to ensure that we get updates. https://docs.renovatebot.com/modules/datasource/git-refs/ should be usable for that, no idea if the hash below would need to be updated as well. (If yes https://github.com/cloudfoundry/app-autoscaler-release/blob/main/.github/workflows/dependency-update.go-mod-tidy_and_make-package-specs.yaml could an approach).

Reply:
The hash will need the update as well. The usual approach to update it is (unfortunately) manual. We would need an automation that may use the instructions on: <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-sources>

Reply:
Good point! 

I will try to onboard renovate here.

 Also, I will try to get away from the commit-hash in `rev` and instead, use the git tag (e.g. `v9.0.17`)

Reply:
@joergdw & @silvestre do you think onboarding renovate / introducing an automation to bump this is worth the effort?